<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="61b0afd8-6fcf-436f-9f8a-032fda521b4e" name="chadillac@CHADILLAC-PC 2012-02-23 12:18:17" runUser="chadillac-PC\chadillac" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="699621a8-4321-403e-843d-a1c2dba0af3c">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="chadillac_CHADILLAC-PC 2012-02-23 12_18_17">
      <DeploymentItem filename="C:\dev\pivotal-csharp-wrapper\PivotalTrackerAPIClient\bin\Debug\PivotalTrackerAPIClient.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-02-23T12:18:17.0791015-06:00" queuing="2012-02-23T12:18:18.0732421-06:00" start="2012-02-23T12:18:18.1699218-06:00" finish="2012-02-23T12:22:22.3349609-06:00" />
  <ResultSummary outcome="Error">
    <Counters total="1" executed="1" error="0" failed="0" timeout="0" aborted="1" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="CHADILLAC-PC" outcome="Error" timestamp="2012-02-23T12:22:22.3339843-06:00">
        <Text>Test host process exited unexpectedly.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="integration_authentication_check_guid" storage="c:\dev\pivotal-csharp-wrapper\pivotaltrackerapiclient.tests\bin\debug\pivotaltrackerapiclient.tests.dll" id="3cb1e635-4037-84b8-ddc7-ff7ac9d4d99d">
      <Execution id="0810c377-788c-4fa4-b714-8ae98cafd442" />
      <TestMethod codeBase="C:/dev/pivotal-csharp-wrapper/PivotalTrackerAPIClient.Tests/bin/Debug/PivotalTrackerAPIClient.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="PivotalTrackerAPIClient.Tests.Integration.Authentication, PivotalTrackerAPIClient.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="integration_authentication_check_guid" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="3cb1e635-4037-84b8-ddc7-ff7ac9d4d99d" executionId="0810c377-788c-4fa4-b714-8ae98cafd442" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="0810c377-788c-4fa4-b714-8ae98cafd442" testId="3cb1e635-4037-84b8-ddc7-ff7ac9d4d99d" testName="integration_authentication_check_guid" computerName="CHADILLAC-PC" duration="00:04:04.0751953" startTime="2012-02-23T12:18:18.2597656-06:00" endTime="2012-02-23T12:22:22.3349609-06:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Aborted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="0810c377-788c-4fa4-b714-8ae98cafd442">
    </UnitTestResult>
  </Results>
</TestRun>